File: src/tests/AlunoTest.php
<?php 

namespace Tests;

use PhpUnit\Framework\TestCase;
use App\Aluno;

class AlunoTest extends TestCase
{
    
    /** @test */
    public function test_calcular_media_escolar()
    {
        $aluno = new Aluno(1, "John ");
        $aluno->addNotaEscolar(5.0);
        $aluno->addNotaEscolar(10.0);

        $media = $aluno->calcularMediaEscolar();

        $this->assertEquals(7.5, $media);
    }

    /** @test */
    public function test_calcular_media_escolar_com_notas_vazias()
    {
        $aluno = new Aluno(1, "John ");
        $media = $aluno->calcularMediaEscolar();
        $this->assertEquals(0.0, $media);
    }
}

File: 1-modelagem-codigo-python/tests/aluno_test.py
from app.aluno import Aluno


def test_calcular_media_escolar():
    aluno = Aluno(1, "John")
    aluno.add_nota_escolar(5.0)
    aluno.add_nota_escolar(10.0)
    media = aluno.calcular_media_escolar()
    assert media == 7.5


def test_calcular_media_escolar_com_notas_vazias():
    aluno = Aluno(1, "John")
    media = aluno.calcular_media_escolar()
    assert media == 0.0